src/sgame/sg_bot.h

```diff
@@ -107,14 +109,16 @@ struct botMemory_t
 	int         futureAimTime;
 	int         futureAimTimeInterval;
 	vec3_t      futureAim;
-	usercmd_t   cmdBuffer;
 	botNavCmd_t nav;
 
 	int lastThink;
 	int stuckTime;
 	vec3_t stuckPosition;
 
 	int spawnTime;
+	//avoid relying on buttons to remember what AI was doing
```

Comment:
I think the intent was to make the fields private, so maybe something like this?

```suggestion

friend void G_BotThink( gentity_t *self );

private:
	// avoid relying on buttons to remember what AI was doing
```

Reply:
Nah no private fields in mega structs please. Also these should not use an `m_` prefix since none of the other members of this struct do that.

Reply:
well, on my side of the code, I did have this private indeed, and the prefix is an heritage of that, I'll fix that.